doc/classes/float.xml

```diff
@@ -221,6 +259,8 @@
 			<argument index="0" name="right" type="float">
 			</argument>
 			<description>
+				Returns [code]true[/code] if both floats are equal.
+				[b]Note:[/b] Due to floating-point number precision errors, consider using [method @GlobalScope.is_equal_approx] instead, which is more reliable.
```

Comment:
```suggestion
				Returns [code]true[/code] if both floats are exactly equal.
				[b]Note:[/b] Due to floating-point precision errors, consider using [method @GlobalScope.is_equal_approx] instead, which is more reliable.
```

I don't think that "number" is necessary here. "Floating-point precision error" or just "Floating-point error" are fairly common terms. Also added "exactly" to the first sentence, I think it helps.